pkgs/development/lua-modules/updater/updater.py

```diff
@@ -201,7 +196,7 @@ def generate_pkg_nix(plug: LuaPlugin):
 
     log.debug("running %s", " ".join(cmd))
 
-    output = subprocess.check_output(cmd, env=custom_env, text=True)
+    output = subprocess.check_output(cmd, text=True)
```

Comment:
suggestion: Not directly related to this PR. I just ran the updater to test it and it failed with a large stack trace (because `luarocks-nix` failed with a non-zero exit code).
Maybe we should add some error handling here, and output the package that caused the error + the luarocks output.

Reply:
Thanks for running it, I think it failed with errors that I solved with the luarocks-nix update in staging (not in master yet). I ran the script out of tree for `luarocks-site` and it worked fine. I am not sure how we can improve the error reporting there, the errors can be very technical and different, most likely coming from a bug in luarocks-nix. Next time you see such an error that you feel can be improved, open an issue.